xml/entity-decl.ent

```diff
@@ -125,6 +128,7 @@
 <!ENTITY ais            "OpenAIS">
 <!ENTITY corosync       "Corosync">
 <!ENTITY pace           "Pacemaker">
+<!ENTITY pmrm           "pacemaker_remote">
```

Comment:
Maybe this should be pacemaker_remoted, I'm not sure.  The problem is that the systemd service is called `pacemaker_remote`, but the process name is `pacemaker_remoted`.  Yay for consistency! :-/